src/index.ts

```diff
@@ -117,7 +108,7 @@ const migrationInit = async function () {
   MigrationPlacesService.prototype._client = placesClient;
   MigrationSearchBox.prototype._client = placesClient;
   MigrationDirectionsService.prototype._client = routesClient;
-  MigrationDistanceMatrixService.prototype._client = clientV1;
+  MigrationDistanceMatrixService.prototype._client = routesClient;
   MigrationDistanceMatrixService.prototype._routeCalculatorName = routeCalculatorName;
```

Comment:
Remove the route calculator name as well since it is now unused (remove here and in the class itself)